addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -277,3 +281,34 @@ def __gather_clearcoat_extension(blender_material, export_settings):
         )
 
     return Extension('KHR_materials_clearcoat', clearcoat_extension, False)
+
+def __gather_transmission_extension(blender_material, export_settings):
+    transmission_enabled = False
+    has_transmission_texture = False
+
+    transmission_extension = {}
+    transmission_slots = ()
+
+    transmission_socket = gltf2_blender_get.get_socket_or_texture_slot(blender_material, 'Transmission')
```

Comment:
`get_socket_or_texture_slot` has been renamed `get_socket` in master.

Reply:
Rebased and updated, thanks!